### ISSUE TITLE
ResBlock中数值问题

### ISSUE THREAD

**Issue by zhaowenlovehome** [HolmesShuan/OISR-PyTorch#5]
论文中图片中你提出的block都是固定值，为什么代码不是固定值，而是可学习的呢
self.scale1 = nn.Parameter(torch.FloatTensor([0.5]), requires_grad=True)
参数requires_grad=Ture 是自动学习权重吗？

**Comment by HolmesShuan** [comment]
Thanks for your comments. The simple answer is `Yes`.
We use the proposed scaling factor as the initialization and fine-tune the parameters via small learning rate.

**Comment by zhaowenlovehome** [comment]
Thanks for your anwser.
Do you have tried constant value? if yes,can you tell the performance!

**Comment by HolmesShuan** [comment]
I'm sorry I couldn't be more helpful. We did not test the constant value.

**Comment by zhaowenlovehome** [comment]
I see，thank you very much
